src/scripts/main.js

```diff
@@ -42,4 +43,8 @@ btns.forEach((btn) => {
     e.target.setAttribute('aria-label', 'Copy to clipboard');
     e.target.blur();
   });
+
+  btn.addEventListener('click', (e) => {
+    e.preventDefault()
+  });
```

Comment:
Hi @zenorocha, on http://www.html5gamedevs.com/ and https://forum.imasters.com.br/ after click copy button the page reloads.